corehq/apps/data_interfaces/models.py

```diff
@@ -73,36 +73,31 @@ def get_case_ids(cls, domain, case_type, boundary_date=None):
         Retrieves the case ids in chunks, yielding a list of case ids each time
         until there are none left.
         """
-        offset = 0
-        chunk_size = 10000
+        chunk_size = 100
 
-        # We have to sort on _uid; _id is not sortable. See:
-        # https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-uid-field.html
-        # https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-id-field.html
+        query = (CaseES()
+                 .domain(domain)
+                 .case_type(case_type)
+                 .is_closed(closed=False)
+                 .exclude_source()
+                 .size(chunk_size))
 
-        while True:
-            query = (CaseES()
-                     .domain(domain)
-                     .case_type(case_type)
-                     .is_closed(closed=False)
-                     .exclude_source()
-                     .sort('_uid')
-                     .start(offset)
-                     .size(chunk_size))
+        if boundary_date is not None:
+            query = query.server_modified_range(lte=boundary_date)
 
-            if boundary_date is not None:
-                query = query.server_modified_range(lte=boundary_date)
+        result = []
 
-            results = query.run()
-            ids = results.doc_ids
+        for case_id in query.scroll():
```

Comment:
so there's a few things going on here - we get these case ids to process rules against the cases, and so whatever the chunk size is we need to be able to process them all within the scroll timeout otherwise the next scroll query will fail.  since the iter_cases methods on CaseAccessors only do 100 cases at a time anyway, i figured it's a good place to start.

also i had first started off with using .run().doc_ids but that was timing out for domains with large numbers of cases so i had to take this approach.